packages/auth0_server_python/src/auth0_server_python/auth_server/server_client.py

```diff
@@ -108,48 +111,50 @@ async def _fetch_oidc_metadata(self, domain: str) -> dict:
             response.raise_for_status()
             return response.json()
 
-    
     async def start_interactive_login(
         self,
         options: Optional[StartInteractiveLoginOptions] = None,
         store_options: dict = None
     ) -> str:
         """
         Starts the interactive login process and returns a URL to redirect to.
-        
+
         Args:
             options: Configuration options for the login process
-            
+
         Returns:
             Authorization URL to redirect the user to
         """
         options = options or StartInteractiveLoginOptions()
-        
+
         # Get effective authorization params (merge defaults with provided ones)
         auth_params = dict(self._default_authorization_params)
         if options.authorization_params:
-            auth_params.update(options.authorization_params)
-            
+            auth_params.update(
+                {k: v for k, v in options.authorization_params.items(
+                ) if k not in INTERNAL_AUTHORIZE_PARAMS}
+            )
```

Comment:
☝️  2/2 relevant changes on this file.
We make sure that authorization params is only updated with anything BUT our internal authorize params.